docs/proposals/mvp-dashboard-design.md

```diff
@@ -10,16 +10,37 @@ Last updated: 2017-01-12
 
 ## Abstract
 
-A short summary of the proposal.
+We propose a dashboard/server that allows for the automated retrieval,
+archive, format, and display of performance tests run in the performance lab.
+This tool will allow our team to investigate performance issues and facilitate
+the sharing of lab results with other teams.
 
 ## Background
 
-An introduction of the necessary background and the problem being solved by
-the proposed change.
+Viewing performance test results from the performance lab as it trends across
+time is tedious and repetitive. To view this data, the following actions must
+take place:
+
+1. Run Performance Test
+1. Retrieve report file
+1. Format data so that it can be entered into a database
+1. Insert data into database
+1. Query the database for timerange of tests
+1. Display the data in a format that is easy to understand (graphs)
+1. Share the visualizations of the tests with other developers
+
+This set of actions occurs everytime there is a need to investigate lab
+performance. Automating these actions allows more time for developers to
+invest in development and investigation of performance issues.
```

Comment:
We should emphasize that our current approach has a very long feedback cycle. By the time, we collect the results and find issues, it's too late to address them in the current release. So, having a system that automatically displays the results as they are gathered, will help us shortened that feedback cycle and catch more issues **early**